core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -60,6 +62,15 @@ oppia.factory('SimpleEditorManagerService', [
       return 'Question ' + minimumStateNumber;
     };
 
+    // It overwrites values so that they match properties of a terminal state.
```

Comment:
Overwrites the given state's properties so that they match those of a terminal state.